lms/validation/authentication/_launch_params.py

```diff
@@ -30,6 +30,10 @@ class LaunchParamsAuthSchema(PyramidRequestSchema):
 
     user_id = marshmallow.fields.Str(required=True)
     roles = marshmallow.fields.Str(required=True)
+    tool_consumer_instance_guid = marshmallow.fields.Str(required=True)
+    lis_person_name_given = marshmallow.fields.Str(missing="")
+    lis_person_name_family = marshmallow.fields.Str(missing="")
+    lis_person_name_full = marshmallow.fields.Str(missing="")
```

Comment:
Adding these to `LaunchParamsAuthSchema` means that when we receive an LTI launch request and deserialize a `request.lti_user` from the verified launch params, that `LTIUser` object will now have the new fields